api/graph/src/main/java/quarks/graph/Connector.java

```diff
@@ -115,5 +115,27 @@ Licensed to the Apache Software Foundation (ASF) under one
      * 
      * @return set of tag values.
      */
-    Set<String> getTags(); 
+    Set<String> getTags();
+    
+    /**
+     * Set the alias for the connector.
+     * <p>
+     * The alias must be unique within the topology.
+     * The alias may be used in various contexts:
+     * <ul>
+     * <li>Runtime control services for the connector are registered with this alias.</li>
```

Comment:
> at the graph level ... services are registered for the Connector's alias
Right, Connectors live as long as the Graph, so can serve as registration object.

+1 for OutputContext.